### PR TITLE
chore: download pkger in update-to-head.sh

### PR DESCRIPTION
**Pull request by jrangelramos**
Adding pkger binary download in order to avoid `pkger` not found error

### CODE REVIEW

File: openshift/release/update-to-head.sh [hunk 1]
```diff
@@ -40,6 +40,7 @@ git checkout upstream/main -B release-next
 # Update openshift's main and take all needed files from there.
 git fetch openshift main
 git checkout openshift/main $custom_files
+go get github.com/markbates/pkger/cmd/pkger
 pkger
```
Comment (dsimansk) [inline openshift/release/update-to-head.sh:44]: Switching L43-44 to the script Markus added might be a better alternative, as it's already leveraging vendored pkger.

https://github.com/knative-sandbox/kn-plugin-func/blob/main/hack/update-pkger.sh

E.g.:
```
./hack/update-pkger.sh
```

Comment (dsimansk) [reply openshift/release/update-to-head.sh:44]: That's just an idea I had when saw the change. (nit picking :))

Comment (lance) [reply openshift/release/update-to-head.sh:44]: It's probably the right approach. Jefferson's fix should be fine for the short term though.

**Pull request merged**